test/trans_interop.py

```diff
@@ -30,7 +90,7 @@ def verify_transaction_amount_confirm(self, nodeTxOut, nodeTxIn, amount):
         nodeTxOut : node Id of the TxOut (debit)
         nodeTxIn : node Id of the TxIn (credit)
         amount : amount to spend in BTC
-    """
+    """ 
```

Comment:
do a "git diff > diff.txt" before you submit a PR and review it for accidental unnecessary changes like the above.

Reply:
thanks